Formula/apache-arrow.rb

```diff
@@ -0,0 +1,36 @@
+class ApacheArrow < Formula
+  desc "Apache Arrow"
+  homepage "https://arrow.apache.org/"
+  url "https://github.com/apache/arrow/archive/apache-arrow-0.4.0.tar.gz"
+  sha256 "0ce9f47e42735d8d48dd83f9ff6a1612c5f2a8e846f3ae23595697e3519dd9b8"
+
+  head "https://github.com/apache/arrow.git"
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+
+  needs :cxx11
+
+  def install
+    ENV.prepend_path "PATH", "/usr/local/bin"
```

Comment:
Strictly speaking, ccache is not required. Because [this problem](https://github.com/apache/arrow/pull/751) occurs, I think that use `depends_on "ccache" => :recommended` now.
What do you think about this idea?